packages/myst-common/src/types.ts

```diff
@@ -78,7 +78,7 @@ export type DirectiveSpec = {
 
 export type RoleSpec = {
   name: string;
-  alias?: string | string[];
+  alias?: string[];
```

Comment:
@fwkoch I removed the ambiguity here. I think this is better, we may need a directive-spec validator in the future that modifies this up front and accepts strings, etc.